api/src/organizational-entities/application/certification-center/certification-center.admin.controller.js

```diff
@@ -25,7 +25,11 @@ const archiveInBatch = async function (request, h) {
 };
 
 const create = async function (request) {
-  const certificationCenter = certificationCenterForAdminSerializer.deserialize(request.payload);
+  const createdBy = extractUserIdFromRequest(request);
+  const certificationCenter = certificationCenterForAdminSerializer.deserialize({
+    data: request.payload.data,
+    createdBy,
+  });
```

Comment:
Suggestion pour améliorer un tout petit peu la compréhension du code : 
```suggestion
  const userId = extractUserIdFromRequest(request);
  const certificationCenter = certificationCenterForAdminSerializer.deserialize({
    data: request.payload.data,
    createdBy: userId,
  });
```